src/utils/GitHelper.ts

```diff
@@ -10,6 +10,22 @@ import Utils from './Utils'
 const exec = util.promisify(childPross.exec)
 
 export default class GitHelper {
+    static #SSH_PATH_RE = new RegExp(
```

Comment:
The common practice across the codebase is to use `private` - rather than `#` prefix. 
It's a minor change, I'll make the change this after merge.